dali/base/dadfs.cpp

```diff
@@ -4698,7 +4726,33 @@ protected: friend class CDistributedFilePart;
         dt.setNow();
         setAccessedTime(dt);
     }
-
+    virtual void addAttrValue(const char *attr, unsigned __int64 value) override
```

Comment:
is look like a copy of same implementation that's already in base class? If so shouldn't be needed